todo.js

```diff
@@ -51,6 +95,35 @@ function deleteTodoItem(todoId) {
   todos = todos.filter((todo) => todo.id !== todoId);
 }
 
-addBtn.addEventListener("click", handleTodo);
+function editTodoItem(todoId) {
+  editTodoId = todoId;
+  renderTodoList();
+}
```

Comment:
Let's handle all the edit functionality here rather than maintaining a global state for `ediTodoId`. 
The flow will be:
1. User clicks on edit todo
2. Handler method is invoked
3. We make the edit state to true of that element
4. We render the ui again

Then when user clicks on cancel or update, similar flow will happen like above